pkg/kwokctl/pki/admin.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
do we need to check in `admin.crt`, `admin.key`, and `ca.crt` if they can be generated by `DumpPki()` in `pki.go`?

Reply:
Is the check format legal?

Reply:
I mean like, do these files need to be part of the PR? Since they are generated.

Reply:
@wzshiming just wanted to clarify if the cert files need to be part of the PR if we can generate them?

Reply:
Got it, I feel that adding an extra dependency on OpenSSL is not good, we can add a Go implementation later.